packages/server-core/src/user/strategies/github.ts

```diff
@@ -47,6 +51,24 @@ export class GithubStrategy extends CustomOAuthStrategy {
     this.app = app
   }
 
+  async createRefreshJob(userId) {
+    const date = await getDateTimeSql()
+    const newJob = await this.app.service(apiJobPath).create({
+      name: '',
+      startTime: date,
+      endTime: date,
+      returnData: '',
+      status: 'pending'
+    })
+
+    const jobBody = await getGithubRepoAccessRefreshJobBody(this.app, newJob.id, userId)
+    await this.app.service(apiJobPath).patch(newJob.id, {
+      name: jobBody.metadata!.name
+    })
+    const jobLabelSelector = `ir-engine/userId=${userId},ir-engine/release=${process.env.RELEASE_NAME},ir-engine/autoUpdate=false`
```

Comment:
What is the use of `,ir-engine/autoUpdate=false` here?

Reply:
It looks like we set ir-engine/autoUpdate to true for when we make a cronjob, which I think only happens if someone sets a project to auto-update. So it's set to false if it's a one-off job.

Reply:
https://github.com/ir-engine/ir-engine/blob/08d42e2d669735dbb0c7521e2199b7627deae097/packages/server-core/src/assets/model-transform/model-transform.helpers.ts#L47
https://github.com/theinfinitereality/irpro-multitenancy/blob/6c41fec421c3d3072c1d424d6726b005dc08b481/services/domain-provision-helper.ts#L136
https://github.com/theinfinitereality/irpro-multitenancy/blob/6c41fec421c3d3072c1d424d6726b005dc08b481/services/account-deletion-helper.ts#L35

I dont see it being set in any of these.

Reply:
I don't think it's necessary, as nothing is specifically looking for that label, it was just added for potential clarity, in case anyone was looking at a job and might benefit from seeing whether it was an auto-updating job. If MT jobs aren't adding that label at all it's fine.